src/NextjsBuild.ts

```diff
@@ -110,7 +110,7 @@ export class NextjsBuild extends Construct {
     const buildEnv = {
       ...process.env,
       [NEXTJS_BUILD_STANDALONE_ENV]: 'true',
-      [NEXTJS_BUILD_OUTPUTTRACEROOT_ENV]: getRootTracePath(nextjsPath),
+      [NEXTJS_BUILD_OUTPUTTRACEROOT_ENV]: path.resolve(),
```

Comment:
curious if this works!

Reply:
My app successfully deploys and functions with this. I'll test it on my other monorepo project after this gets merged.